TypeScript.YAML-tmLanguage

```diff
@@ -495,16 +675,16 @@ repository:
     - include: '#expression'
 
   expression-operator:
-    name: keyword.others.ts
-    match: \b(delete|in|instanceof|new|typeof|as|is|of)\b
+    name: keyword.operator.ts
```

Comment:
Ok, I see, we try our best to associate 'operator' with actual operators (mathematical or logical). So I'd suggest reverting this change.